conda/testing/helpers.py

```diff
@@ -220,12 +220,14 @@ def _export_subdir_data_to_repodata(subdir_data: SubdirData):
     packages = {}
     packages_conda = {}
     for pkg in subdir_data.iter_records():
+        if pkg.timestamp:
+            # ensure timestamp is dumped as int in milliseconds
+            # (pkg.timestamp is a kept as a float in seconds)
```

Comment:
I bet we use time.time() to get the float timestamp in conda-build as well? Assuming it would be hard to switch to ints everywhere?